### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by benyamin-noori**
Fix lint problems. 
Add `npm run test` to travis CI. 
Fix tests that were failing.

### CODE REVIEW

File: components/d2l-sequences-content-link.html [hunk 1]
```diff
@@ -44,26 +44,26 @@
 						type: String,
 						computed: '_getLinkLocation(entity)'
 					}
-				}
+				};
 			}
 			static get observers() {
-				return ['_render(entity)']
+				return ['_render()'];
```
Comment (jwong-d2l) [inline components/d2l-sequences-content-link.html]: what happens if we take out the thing that is being watched? @jbydeley-d2l

File: locales.json [hunk 2]
```diff
@@ -19,10 +19,13 @@
 		"activitiesFinishedGreatJob": "You have completed all of the activities in this lesson. Great job!",
 		"imDone": "I'm Done",
 		"niceWork": "Nice work so far!",
-		"youMayHave": "You may have",
+		"you": "You",
```
Comment (gravindran-d2l) [inline locales.json]: Could you verify this one -- if master had "youMayHave", you might be overwriting something.

File: components/d2l-sequences-content-link.html (hunk 1 above)
Comment (jwong-d2l) [reply components/d2l-sequences-content-link.html]: aside from this one thing, I think it looks good!

File: locales.json (hunk 2 above)
Comment (benyamin-noori) [reply locales.json]: Actually I have no idea how this change happened. It looks like it should be "youMayHave" though. I'll change it back.

Comment (jwong-d2l) [reply locales.json]: these are in master, so maybe it's just the merge showing a false alarm? https://github.com/BrightspaceHypermediaComponents/sequences/blob/master/locales.json

Comment (jwong-d2l) [reply locales.json]: oh wait, the "you" one is different.

Comment (benyamin-noori) [reply locales.json]: I talked to @tinglisD2L and she says it should be "you" and not "youMayHave". So what we have here is the correct version.

Comment (gravindran-d2l) [reply locales.json]: Excellent.  Code review FTW!

File: components/d2l-sequences-content-link.html (hunk 1 above)
Comment (jbydeley-d2l) [reply components/d2l-sequences-content-link.html]: Polymer uses that to know when to trigger the `_render` function. We need to have `entity` in there or it won't fire when `entity` changes.

Comment (benyamin-noori) [reply components/d2l-sequences-content-link.html]: Added a comment to ignore no-unused-vars for this file specifically. I thought we'd still want to have this rule for the rest of the project.

**Pull request merged**